modules/ichorcna/1.0/ichorcna.smk

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env snakemake
+
+
+# ---------------------------------------------------------------------------- #
+##### ATTRIBUTION #####
+# ---------------------------------------------------------------------------- #
+
+# Original snakemake author: Jasper Wong
+# Module author: Jasper Wong
+# Additional contributors: N/A
+
+
+# ---------------------------------------------------------------------------- #
+##### SETUP #####
+# ---------------------------------------------------------------------------- #
+
+### Modules ###
+
+import pandas as pd
+import numpy as np
+import oncopipe as op
+import glob
+import os
+
+### Directories ###
+
+resultsDir = "results/"
+logDir = "log/"
+
+
+# Setup module and store module-specific configuration in `CFG`.
+CFG = op.setup_module(
+    name = "ichorcna", 
+    version = "1.0",
+    subdirectories = ["inputs", "readDepth", "seg", "outputs"]
+)
+
+localrules:
+    _ichorcna_input_bam,
+    _ichorcna_output,
+    _ichorcna_all
+
+# ---------------------------------------------------------------------------- #
+##### RULES #####
+# ---------------------------------------------------------------------------- #
+
+CFG["runs"]["binSize"] = str(CFG["options"]["readcounter"]["binSize"])
+
+# Symlinks the input files into the module results directory (under '00-inputs/')
+rule _ichorcna_input_bam:
+    input:
+        bam = CFG["inputs"]["sample_bam"],
+        bai = CFG["inputs"]["sample_bai"]
+    output:
+        bam = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam",
+        bai = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam.bai", # specific to readCounter
+        crai = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam.crai" 
+    run:
+        op.relative_symlink(input.bam, output.bam)
```

Comment:
we started to use the `op.absolute_symlink` here instead of relative symlink. This will also need the check for oncopipe version, and you can find example in one of the recent modules (battenberg/1.1, pathseq/1.0 etc)

Reply:
Changed bam inputs to op.absolute_symlink